System/Constants.h

```diff
@@ -22,6 +22,10 @@ namespace RTE {
 	static constexpr int c_DefaultAtomGroupResolution = 5; //!< The global default AtomGroup resolution setting.
 #pragma endregion
 
+#pragma region AI Constants
+    static constexpr float c_PathFindingDefaultDigStrength = 35.0F; // A default penetration value that'll allow us to move through corpses and stuff
```

Comment:
I know this is non-obvious, but for inline comments we use `//!< comment here` because it allows doxygen to autogenerate properly. Also, don't forget the `.` at the end :)